.github/workflows/well-known-environment.yml

```diff
@@ -145,7 +145,7 @@ jobs:
       - name: Configure AWS credentials for account
         uses: ./.github/actions/configure-aws-credentials
         with:
-          environment-name: $TARGET_ACCOUNT_NAME
```

Comment:
Can't use env vars straight in here they would have to be in a `run` command doh

Reply:
@A-Ashiq i can't remember the exact syntax, but maybe try something like this:

```
environment-name: {{ env.TARGET_ACCOUNT_NAME }}
```